src/app/core/admin/admin-overview/admin-overview.component.html

```diff
@@ -5,89 +5,97 @@ <h1>Administration & Configuration</h1>
   you know what you are doing.
 </p>
 
-<hr />
+<mat-accordion class="admin-sections margin-top-large" multi>
+  @for (section of adminOverviewService.sections; track section.id) {
+    <!-- Hide subscription section if not in SaaS environment -->
+    @if (section.id !== "subscription" || isSaasEnvironment()) {
+      <mat-expansion-panel
+        [expanded]="section.expanded"
+        (expandedChange)="onSectionExpandedChange(section.id, $event)"
+      >
+        <mat-expansion-panel-header>
+          <mat-panel-title>{{ section.title }}</mat-panel-title>
+          <mat-panel-description>
+            {{ section.description }}
```

Comment:
I don't see the sections themselves becoming dynamic and configured. Only items within a section can be added by various modules.

In my opinion, the template will be easier to understand if we hard-code the expansion panels here in the template and then only within some panels (mostly the "templates" section) we dynamically add configured items.

Other sections, like the Technical section, can even keep hard-coded entries, because this is only core functionality for maintenance tasks.